app/webpacker/components/Tickets/TicketWorkbenches/CompetitionResultActionerView/CreateWcaIds.jsx

```diff
@@ -19,9 +22,34 @@ export default function CreateWcaIds({ ticketDetails }) {
     queryKey: ['inbox-person-summary', id],
     queryFn: () => getInboxPersonSummary({ ticketId: id }),
   });
+  const {
+    mutate: deleteInboxPersonsMutate,
+    isPending,
+    isError: isDeleteError,
+    error: deleteError,
+  } = useMutation({
+    mutationFn: deleteInboxPersons,
+    onSuccess: () => {
+      queryClient.setQueryData(
+        ['ticket-details', id],
```

Comment:
I'm seeing a lot of this deeply nested `setQueryData` for the ticket status. Perhaps it's time to move this into a util or lib function somewhere?

Reply:
Ya sounds like a good idea. I'll come up with a solution in one of the upcoming PRs (because it will be out of scope of this PR). Thanks for approving.